handbook/engineering/campaigns.md

```diff
@@ -0,0 +1,23 @@
+# Campaigns
+
+User-facing documentation: https://docs.sourcegraph.com/user/campaigns
+Developer documentation: https://docs.sourcegraph.com/dev/campaigns_development
+
+## Mission Statement
+
+Campaigns are to large-scale code changes what Netlify is to static site generation and AWS Lambda is to handling HTTP requests.
```

Comment:
like it, but `large-scale code changes` would omit our second proposed use case that doesn't generate `code-changes`. I'm not against this though, just a side-note

Reply:
Good point! In one sense this use case can be part of the list: users don't have to worry about their changes staying up to date with ever updating code.

The other sense of that use case is still under-defined, so I don't think it's necessary to include it _yet_. WDYT?

Reply:
absolutely, just wanted to kick that thought off so we are on the same page that it might be a use-case that campaigns will pick up in the future :)

Reply:
Added another point!